browser/net/brave_site_hacks_network_delegate_helper.cc

```diff
@@ -103,39 +46,19 @@ bool ApplyPotentialReferrerBlock(net::URLRequest* request) {
   return false;
 }
 
+}  // namespace
+
+namespace brave {
+
 int OnBeforeURLRequest_SiteHacksWork(
     net::URLRequest* request,
     GURL* new_url,
     const ResponseCallback& next_callback,
     std::shared_ptr<BraveRequestInfo> ctx) {
-  const GURL& url = request->url();
-
-  if (IsEmptyDataURLRedirect(url)) {
-    *new_url = GURL(kEmptyDataURI);
-    return net::OK;
-  }
-
-  if (IsBlockedResource(url)) {
-    request->Cancel();
```

Comment:
request->Cancel() not needed when you return net::ERR_ABORTED and actually it causes a debug assertion if you do it.